diff --git a/make/dll.mak b/make/dll.mak
index 7895c754..cfda8c26 100644
--- a/make/dll.mak
+++ b/make/dll.mak
@@ -55,25 +55,21 @@ BINDIR = bin/$(ARCH)
 # Mac OS X on Intel 32 bit processor
 ifeq ($(ARCH),maci)
 DLL_SUFFIX := dylib
-DLL_LDFLAGS += -m32
 endif
 
 # Mac OS X on Intel 64 bit processor
 ifeq ($(ARCH),maci64)
 DLL_SUFFIX := dylib
-DLL_LDFLAGS += -m64
 endif
 
 # Linux-32
 ifeq ($(ARCH),glnx86)
 DLL_SUFFIX := so
-DLL_LDFLAGS += -m32
 endif
 
 # Linux-64
 ifeq ($(ARCH),glnxa64)
 DLL_SUFFIX := so
-DLL_LDFLAGS += -m64
 endif
 
 # --------------------------------------------------------------------
